### DISCUSSION TITLE
Can someone tell me what english font it is in the screenshot on the koreader wiki page?

### DISCUSSION THREAD

**Discussion by Joey1986** [koreader/koreader#10410] [Q&A] [answered]
Thanks very much!
[https://github.com/koreader/koreader/wiki/Dictionary-support#dictionary-lookups-in-scanned-pages](url)
![dictionary_lookup](https://github.com/koreader/koreader/assets/20594209/58a4bdc7-832e-41a4-b569-a83dfc3f7be4)

**Comment by Frenzie** [comment]
Are you referring to Noto?

**Comment by Joey1986** [comment]
No, I don't think it's Noto.
I mean which font the English text is on the screen shot.  Not the English dictionary popup.






-------- 原始邮件 --------
发件人：Frans de Jonge ***@***.***>
时间：2023年5月10日 16:56
收件人：koreader/koreader ***@***.***>
抄送：Joey1986 ***@***.***>,Author ***@***.***>
主题：Re: [koreader/koreader] Can someone tell me what english font it is in the screenshot on the koreader wiki page? (Discussion #10410)


Are you referring to Noto?

**Reply by Frenzie** [reply]
I'm not sure what that one is.

**Comment by gingerbeardman** [comment, marked as answer]
@Joey1986 the font is _Fontin Regular_

- https://www.whatfontis.com/FF_Fontin-Regular.font?text=But%20now%20and%20again%20there#ct
- https://www.cdnfonts.com/fontin.font
- https://fontsgeek.com/fonts/Fontin-Regular

![image](https://github.com/koreader/koreader/assets/49612/df54ccd7-02e5-4fba-b325-0936e5fe744d)

**Reply by NiLuJe** [reply]
Thanks, I knew I was familiar with it, but I couldn't remember its name ;o).
